frontend/src/components/pages/FridgeManagement/FridgeCheckIns/DeleteCheckIns.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+  Button,
+  Container,
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  HStack,
+  Input,
+  Text,
+  useToast,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { useForm } from "react-hooks-helper";
+import DatePicker, { DateObject } from "react-multi-date-picker";
+import { useHistory } from "react-router-dom";
+
+import CheckInAPIClient from "../../../../APIClients/CheckInAPIClient";
+import * as Routes from "../../../../constants/Routes";
+import { CheckIn } from "../../../../types/CheckInTypes";
+import ErrorMessages from "./ErrorMessages";
+
+const checkInDefaultData = {
+  startDate: "",
+  endDate: "",
+} as CheckIn;
+
+const DeleteCheckInsPage = () => {
+  const toast = useToast();
+  const history = useHistory();
+  const [checkInFormValues, setCheckInForm] = useForm(checkInDefaultData);
+  const [dateRange, setDateRange] = useState<DateObject[]>([
+    new DateObject(),
+    new DateObject().add(1, "days"),
+  ]);
+  const [formErrors, setFormErrors] = useState({
+    dateRange: "",
+  });
+
+  useEffect(() => {
+    setCheckInForm({
+      target: { name: "startDate", value: new DateObject().toString() },
+    });
+    setCheckInForm({
+      target: {
+        name: "endDate",
+        value: new DateObject().add(1, "days").toString(),
+      },
+    });
+  }, []);
+
+  const handleDateRangeChange = (e: DateObject[]) => {
+    if (e[0]) {
+      setCheckInForm({
+        target: { name: "startDate", value: e[0].toString() },
+      });
+    }
+    if (e[1]) {
+      setCheckInForm({
+        target: { name: "endDate", value: new DateObject(e[1]).add(1, "days").toString() },
```

Comment:
seems to work with this little hack - not sure if the issue is with `deleteCheckInsByDateRange` or smth else